cargo-bpf/src/llvm.rs

```diff
@@ -1,3 +1,14 @@
+cfg_if::cfg_if! {
```

Comment:
I think this is probably a more robust way of doing this. I do really like that it throws an error if there's an incompatible configuration, though!

```rust
#[cfg(feature = "llvm-sys-110")]
use llvm_sys_110 as llvm_sys;
#[cfg(feature = "llvm-sys-120")]
use llvm_sys_120 as llvm_sys;

if #[cfg(feature = "llvm11")] {
        #[rustversion::since(1.52)]
        compile_error!("Can not use LLVM11 with Rust >= 1.52");
}
```

Reply:
Ok I reflected your suggestion to the code